internal/dag/builder_test.go

```diff
@@ -104,154 +105,185 @@ func TestDAGInsert(t *testing.T) {
 		},
 	}
 
-	i1 := &v1beta1.Ingress{
+	i1V1 := &networking_v1.Ingress{
```

Comment:
this file's diff is pretty impossible to review - can you describe the changes? I think you said you were creating a second copy of all the Ingress tests, that used a `v1` Ingress instead of a `v1beta1`, so we'd have tests for both the `v1beta1` and the `v1` representations.

Reply:
Yeah it's unfortunate that the diff is so hard to read, but I copied all the v1beta1 ingress resources into networking_v1 ingress resources and corresponding tests so that we could test the v1beta1 resources along with the networking_v1 resources.

In the next set of PRs, we should expand these tests with the new features in v1 (e.g. ingressclass, path type, etc)